docs/minio-limits.md

```diff
@@ -1,56 +1,64 @@
-- BucketVersions, BucketVersioning (Use [`s3git`](https://github.com/s3git/s3git))
-- BucketWebsite (Use [`caddy`](https://github.com/mholt/caddy) or [`nginx`](https://www.nginx.com/resources/wiki/))
-- BucketAnalytics, BucketMetrics, BucketLogging (Use [bucket notification](https://docs.minio.io/docs/minio-client-complete-guide#events) APIs)
-- BucketRequestPayment
-- BucketTagging
-
-#### List of Amazon S3 Object API's not supported on Minio
-
-- ObjectACL (Use [bucket policies](https://docs.minio.io/docs/minio-client-complete-guide#policy) instead)
-- ObjectTorrent
-- ObjectVersions
-
-### Object name restrictions on Minio
-Object names that contain characters `^*|\/&";` are unsupported on Windows and other file systems which do not support filenames with these characters. Note that this list is not exhaustive, and depends on the maintainers of the filesystem itself.
+|Part size|5 MB to 5 GB. The last part can be 0 B to 5 GB.|
+|Maximum number of parts returned per list in a request for parts| 1000|
+|Maximum number of objects returned per list in a request for objects| 1000|
+|Maximum number of multi-part uploads returned per list in a request for multi-part uploads| 1000|
+
+### 4. <a name="list-of-amazon-s3-apis-not-supported-on-Minio"></a>List of Amazon S3 APIs Not Supported on Minio
+The following APIs are considered redundant or less useful outside of AWS S3. If there are additional APIs that should be added to this list, please open a [github issue](https://github.com/minio/minio/issues).
+
+#### 4.1 List of Amazon S3 Bucket APIs Not Supported on Minio:
+
+- `BucketACL` (use [bucket policies](https://docs.minio.io/docs/minio-client-complete-guide#policy) instead)
+- `BucketCORS` (CORS is enabled by default on all buckets for all HTTP verbs)
+- `BucketLifecycle` (not required for the Minio erasure-coded backend)
+- `BucketReplication` (use [`mc mirror`](https://docs.minio.io/docs/minio-client-complete-guide#mirror) instead)
+- `BucketVersions`, `BucketVersioning` (use [`s3git`](https://github.com/s3git/s3git))
+- `BucketWebsite` (use [`caddy`](https://github.com/mholt/caddy) or [`nginx`](https://www.nginx.com/resources/wiki/))
+- `BucketAnalytics`, `BucketMetrics`, `BucketLogging` (use [bucket notification](https://docs.minio.io/docs/minio-client-complete-guide#events) APIs)
+- `BucketRequestPayment`
+- `BucketTagging`
+
+#### 4.2 List of Amazon S3 Object APIs Not Supported on Minio:
+
+- `ObjectACL` (use [bucket policies](https://docs.minio.io/docs/minio-client-complete-guide#policy) instead)
+- `ObjectTorrent`
+- `ObjectVersions`
+
+### 5. <a name="object-name-restrictions"></a>Object Name Restrictions
+Minio does not support object names that contain any of the following characters: `^*|\/&#";`. This restriction is applicable to file systems that do not support filenames containing these characters. There may be additional filename character restrictions on other file systems.
```

Comment:
# works with ntfs and xfs, is there another file system this is known to fail with?